### DISCUSSION TITLE
Can't "select" or "toggle" entries for some pickers. (`lgrep_curbuf` or `blines` are examples)

### DISCUSSION THREAD

**Discussion by yanghu** [ibhagwan/fzf-lua#1171] [Q&A] [answered]
I want to send fzf-lua results to qflist. This works for regular grep pickers (`live_grep_glob`, etc)

However, if I do `lgrep_curbuf` or `blines`, I can't select entries(using `<Tab>`, or use my own keymap set to `toggle-all`), so I can't send multiple entries to qflist.  (I can still send the "current" entry to qflist)

Here's related keymap configs I used if it's helpful:

```lua
	fzf_lua.setup({
		global_resume = true,
		keymap = {
			builtin = {
				["ctrl-u"] = "half-page-up",
				["ctrl-d"] = "half-page-down",
				["<C-f>"] = "preview-page-down",
				["<C-b>"] = "preview-page-up",
				["<C-a>"] = "toggle-all",
			},
			fzf = {
				["ctrl-u"] = "half-page-up",
				["ctrl-d"] = "half-page-down",
				["ctrl-f"] = "preview-page-down",
				["ctrl-b"] = "preview-page-up",
				["ctrl-a"] = "toggle-all",
			},
		},
		actions = {
			files = {
				["ctrl-q"] = actions.file_sel_to_qf,
			},
		},
```

and 

```lua
grep = {
	-- cmd = 'ugrep -RIjnkzs --hidden --ignore-files --exclude-dir=".git"',
	rg_opts = "--column --trim --line-number --no-heading --color=always --smart-case --max-columns=4096 -e",
	path_shorten = 4,
	winopts = bottom_row,
	-- winopts = {
	-- 	fullscreen = true,
	-- },
	formatter = "path.filename_first",
	actions = {
		["ctrl-q"] = actions.file_sel_to_qf,
	},
},
```

**Comment by ibhagwan** [comment, marked as answer]
The default in `blines` is `--no-multi`:
https://github.com/ibhagwan/fzf-lua/blob/cb208cd24bacac5893249ea3a8624d705d2135f9/lua/fzf-lua/defaults.lua#L531

You can enable multiple selection by adding `fzf_opts = { ['--no-multi']=false, ['--multi']=true }` to the `blines` setup section:
```lua
fzf_lua.setup({
  blines = {
    fzf_opts = {
      ['--no-multi']=false,
      ['--multi']=true,
    }
  }
})
```

Since `lgrep_curbuf` and `grep_curbuf` both use the same options as `blines` (since they are all lines in the current buffer) this will save both your issues:
https://github.com/ibhagwan/fzf-lua/blob/cb208cd24bacac5893249ea3a8624d705d2135f9/lua/fzf-lua/providers/grep.lua#L404

**Reply by yanghu** [reply]
Yes it works now. 
 thank you for the quick answer, and for making this amazing plugin!
